frontend/src/pages/EnergyCalculation/EnergyCalculation.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import { TextField, Typography, Button, Container, Paper, Grid } from '@mui/material';
+import Navbar from '../../components/Navbar';
+import Sidebar from '../../components/Sidebar';
+
+const EnergyCalculation = () => {
+  const [amps, setAmps] = useState('');
+  const [volts, setVolts] = useState('');
+  const [hoursPerDay, setHoursPerDay] = useState('');
+  const [costPerKwh, setCostPerKwh] = useState('');
+  const [dailyCost, setDailyCost] = useState(null);
+  const [monthlyCost, setMonthlyCost] = useState(null);
+
+  const calculateCost = () => {
```

Comment:
remove this function and add the middleware part for fetching data from backend 
you can find in routes/ energy Calculator. 
thanks 
good Job!!